exercises/bank-account/canonical-data.json

```diff
@@ -0,0 +1,278 @@
+{
+  "exercise": "bank-account",
+  "comments": [
+    "For this exercise it is possible to implement concurrency tests",
+    "but for language tracks that do not support concurrency, the tests",
+    "are written to be run in a single thread.",
+    "But for language tracks that do support concurrency, so is it possible",
```

Comment:
"is it possible" seems to imply a question, but the sentence here ends with a period instead of a question mark. Was it meant to be a question?

Reply:
Are we talking about the sentence on line 7-8?

Reply:
Since line 4 has a pretty similar wording

Reply:
I think I improved it, can you check again?